nexus/pkg/server/sender.go

```diff
@@ -301,6 +307,37 @@ func (s *Sender) sendHistory(record *service.Record, _ *service.HistoryRecord) {
 	}
 }
 
+func (s *Sender) sendSummary(_ *service.Record, summary *service.SummaryRecord) {
```

Comment:
do we need to do the summary manipulation both in the handler and the sender?
can we get away with only doing it in one place?